data-model/src/store.rs

```diff
@@ -150,6 +150,25 @@ where
 /// Returned when the store chooses to not resume a subscription.
 pub struct ResumptionFailedError(pub u64);
 
+/// Describes which entries to ignore during a query.
+#[derive(Default)]
+pub struct QueryIgnoreParams {
+    /// Omit entries with locally incomplete corresponding payloads.
+    pub ignore_incomplete_payloads: bool,
+    /// Omit entries with an empty payload.
```

Comment:
```suggestion
    /// Omit entries whose payload is the empty string.
```

Just because I still tend to read this as "Ignore entries for which we have no payload data yet" if I turn off my brain. Not really sure whether my suggestion helps with that, though =S

Reply:
I sympathise and accept.